app/src/androidTest/java/com/github/houseorganizer/houseorganizer/FirebaseTestsHelper.java

```diff
@@ -202,6 +202,24 @@ protected static Map<String, Object> fetchHouseholdData(String houseName, Fireba
         return task.getResult().getData();
     }
 
+    protected static List<DocumentSnapshot> fetchHouseholdEvents(String houseName, FirebaseFirestore db) throws ExecutionException, InterruptedException {
+        Task<QuerySnapshot> task = db.collection("events").whereEqualTo("household", houseName).get();
+        Tasks.await(task);
+        return task.getResult().getDocuments();
+    }
+
+    protected static boolean householdExists(String houseName, FirebaseFirestore db) throws ExecutionException, InterruptedException {
+        Task<DocumentSnapshot> task = db.collection("households").document(houseName).get();
+        Tasks.await(task);
+        return task.getResult().exists();
+    }
+
+    protected static boolean EventExists(String event, FirebaseFirestore db) throws ExecutionException, InterruptedException {
```

Comment:
Is there a reason to start the method name with a capital letter here?

Reply:
No, just a typo. I will change it before merging.